servant-js/examples/counter.hs

```diff
@@ -43,7 +43,7 @@ type TestApi = "counter" :> Post '[JSON] Counter -- endpoint for increasing the
           :<|> "counter" :> Get '[JSON] Counter -- endpoint to get the current value
 
 type TestApi' = TestApi
-           :<|> Raw -- used for serving static files
+          :<|> Raw IO Application -- used for serving static files
```

Comment:
@jkarni Have you had much more thought on this? @alpmestan Any thoughts or opinions?